### ISSUE TITLE
Pluggable JS

### ISSUE THREAD

**Issue by subins2000** [krishnanunnir/manglish#3]
Hi

Instead of this being a Trix thing, it'd be great if it's an easy pluggable script like [swanalekha](https://swanalekha.smc.org.in/browser/#swanalekha-javascript-version) that can be binded to any input. Some suggestions

* Use yarn as package manager
* Use a `dist` folder setup to build a single script. [Example](https://github.com/subins2000/p2pt/tree/master/dist). See package.json in it
* Use [standard](https://standardjs.com/)

**Comment by krishnanunnir** [comment]
Yes, I have actually changed the code to be pluggable, but I am not very happy with the implementation. I will try to change it to a cleaner format. 

I will check how to do with yarn, dist, and try to adopt a standard.

@subins2000 , Thanks a lot for the input 😄
